packages/sitecore-jss-angular/src/components/base-field.directive.ts

```diff
@@ -34,12 +35,27 @@ export abstract class BaseFieldDirective {
    */
   protected renderEmpty() {
     if (this.field?.metadata && this.editable) {
+      this.renderMetadataTag('open');
       if (this.emptyFieldEditingTemplate) {
         this.viewContainer.createEmbeddedView(this.emptyFieldEditingTemplate);
       } else {
-        this.viewContainer.clear();
         this.viewContainer.createComponent(this.defaultFieldEditingComponent);
       }
+      this.renderMetadataTag('close');
+    }
+  }
+
+  /**
+   * Renders a metadata chrome marker for the field. Required by Pages in editMode 'metadata'.
+   * @param {string} kind - 'open' or 'close' to indicate the start or end of the metadata chrome
+   */
+  protected renderMetadataTag(kind: 'open' | 'close') {
```

Comment:
@stasmaxymov Regarding 'open' vs 'close' type
I'm just thinking if we need to introduce a enum in a sitecore-jss package? under /editing submodule, since I see we use the inline value pretty often

Reply:
I generally favor union type over enum, does it suits you if I use it?

Reply:
The thing that these string values are passed to this function in other files
So I just wanted to be sure we are not passing anywhere inline hardcoded "string" value, while we can define an enum. Using Union you can just define the type

Reply:
I see, you want to reserve right to change serialization of values from a single place.

Reply:
Can we name it instead as "MetadataKind"? "Chrome" is really confusing since we have "Chromes" mode as well